sources/proc_common.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	numRegexPattern = regexp.MustCompile(`[0-9]*\.[0-9]+|[0-9]+`)
```

Comment:
Correct. If you use a raw string (the "`" notation), you don't have to use a double backslash. Originally, I used double quotes for the string which _does_ require the double backslash.